clocks/hello_48MHz/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ add_executable(hello_48MHz
         hello_48MHz.c
         )
 
-# Pull in our pico_stdlib which pulls in commonly used features
+# pull in common dependencies
```

Comment:
...and additional clocks hardware support?

Reply:
Whoops!